jobs.groovy.j2

```diff
@@ -594,12 +597,13 @@ branches.each {
 {# apply config related to 'cronly_jobs' jobs                                #}
 {%    if job_type == 'cronly_jobs'                                          -%}
 {%        set _job_name = job_name                                          -%}
-{# lets place these jobs inside the 'master' branch folder, as its likely
-   people will be looking into that view when monitoring the master branch   #}
-  job("${dashProject}/master/_{{ _job_name }}") {
+  job("${dashProject}/${branchName}/_{{ _job_name }}") {
+    parameters {
+        textParam("TRIGGERED_BRANCH", "${branchName}", "Branch that triggered this job" )
+    }
     label("{{ job_values.on_nodes_with_labels }}")
```

Comment:
we need to pass the branch name to our vagrant build tutorial box so that it can build the package for the right OS